epm/dist/deb/debian/postinst

```diff
@@ -15,27 +15,6 @@ db_timeout = 10
   max_recv_message_size = 16777216
   max_send_message_size = 16777216
 EOF
-cat << EOF > /etc/systemd/system/epm.service
-[Unit]
-Description=epm
-Documentation=https://inclavare-containers.io
-After=network.target
-
-[Service]
-ExecStart=/usr/local/bin/epm --config /etc/epm/config.toml --stderrthreshold=0
-Restart=always
-RestartSec=5
-Delegate=yes
-KillMode=process
-OOMScoreAdjust=-999
-LimitNOFILE=1048576
-LimitNPROC=infinity
-LimitCORE=infinity
-
-[Install]
-WantedBy=multi-user.target
-EOF
 
 mkdir -p /var/local/epm
-systemctl enable epm
-systemctl start epm
```

Comment:
systemd is standard component so you cannot change it in this way only because you are trying to install it in container without systemd support. 

See how sgx-aesm-service resolves this issue:
```
linux-sgx/linux/installer/rpm/sgx-aesm-service
```

then run build.sh. Note that the tarball is also dynamically created by build.sh.